app/views/content/life-as-a-teacher/teaching-as-a-career/what-its-like-to-be-a-teacher/_article.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+<h2 class="heading--box-blue"> An example day in the life of a teacher</h2>
+ <p>Your school day could vary hugely depending on the age group or subject you teach, the number of pupils in your class and many other factors. However, some things, like the timetable of the day, are more predictable. This example day is designed to give you an idea of the sorts of things you could be doing on an average day.</p>
+ <h3>Morning</h3>
+  <h4>7:30am arrive at school</h4>
+   <p>Teachers often start their day early, arriving at school somewhere between 7:00am to 8:00am. The morning is usually spent preparing resources for the day ahead so that everything is organised for when the pupils arrive.</p>
+  <h4> 8:45am pupils arrive</h4>
+   <p>Depending on your role, you might be responsible for a form group or registering pupils. During this time, pupils typically engage in morning activities such as word puzzles. It's also an opportunity to talk to students individually to check their wellbeing.</p>
+    <%= render 'content/shared/quotes/quote_secondary_phil' %>
+  <h4>9:15am morning lessons</h4>
+   <p>The structure of the school day will depend on your school, but often there are 2 lessons taught in the morning followed by a break. In total, there may be 5 hour-long lessons in the day. Each class could be around 30 pupils and will have its own dynamics as well as unique needs.</p>
+  <h4>11:15am morning break</h4>
+   <p>Teachers are sometimes responsible for break-time duties. This involves supervising pupils for their safety but it's also an opportunity to interact with pupils outside of the classroom and build relationships.</p>
+  <h4>11:35am lessons</h4>
+   <p>After break, pupils return to lessons with renewed energy, which can be harnessed and channeled into their work. During this time, teachers usually teach 2 more lessons before lunch.</p>
+   <%= render 'content/shared/quotes/quote_science_jessica' %>
+  <h4>Lunch</h4>
+   <p>Lunch is an opportunity for teachers to have a well-earned break and go to the staff room to talk about how the morning has been. Many teachers also use this as an opportunity to get ahead on their marking. By tackling some books during this time, it can reduce the amount of marking needed to be done in the afternoon.</p>
+   <%= render 'content/shared/quotes/quote_secondary_mike' %>
+ <h3>Afternoon</h3>
+  <h4>2:15pm afternoon lessons</h4>
+   <p>After lunch, there's often just one more lesson to teach. Schools sometimes work the timetable this way because after lunch pupils are generally more excitable making it harder to teach.</p>
+  <h4>3:15pm pupils leave school</h4>
+   <p>When pupils leave school the work continues. Teachers generally spend this time marking work and planning upcoming lessons.</p>
```

Comment:
Is there a way we can make this sound a bit more positive? I read 'the work continues' as a bit gloomy, but could just be me!